debugview/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    alias libs.plugins.android.library
+    alias libs.plugins.kotlin.android
+}
+
+if (!project.getProperties()["ANDROID_VARIANT_TO_PUBLISH"].contains("customEntitlementComputation")) {
+    apply plugin: "com.vanniktech.maven.publish"
+}
+
+apply from: "$rootProject.projectDir/library.gradle"
+
+android {
+    namespace 'com.revenuecat.purchases.debugview'
+
+    flavorDimensions = ["apis"]
+    productFlavors {
+        defaults {
+            dimension "apis"
+        }
+    }
+
+    defaultConfig {
+        minSdkVersion 21 // Compose requires minSdkVersion 21
+    }
+
+    buildFeatures {
+        compose true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion '1.4.0-alpha02'
```

Comment:
The compiler version needs to match the kotlin version https://developer.android.com/jetpack/androidx/releases/compose-kotlin. And for 1.7.21, it needs to be `1.4.0-alpha02`. It won't work with `1.4.0`. The alternative here would be to downgrade to 1.7.20, which would match with compiler version 1.3.2... As for upgrading to kotlin 1.8, it would cause breaking changes, since it seems apps using kotlin 1.6 aren't really compatible due to some dependencies. Open to suggestions here...